app.json

```diff
@@ -33,7 +33,7 @@
       "description": "GitHub Integration ID"
     },
     "GITHUB_INTEGRATION_PEM": {
-      "description": "GitHub Integration private key (base64-encoded)"
+      "description": "GitHub Integration private key (download the file, encode it with base64, then paste the result here)"
```

Comment:
A PEM file has newlines in it, but a Heroku environment variable cannot have newlines in it.